src/main/java/com/crowdin/client/glossaries/GlossariesApi.java

```diff
@@ -11,7 +11,26 @@
 import com.crowdin.client.core.model.PatchRequest;
 import com.crowdin.client.core.model.ResponseList;
 import com.crowdin.client.core.model.ResponseObject;
-import com.crowdin.client.glossaries.model.*;
```

Comment:
why not keep this instead of the explicit imports below?

Reply:
Using a wildcard imports is problematic for various reasons. 
Let me point you to [some link](https://stackoverflow.com/questions/147454/why-is-using-a-wild-card-with-a-java-import-statement-bad).

But in case you disagree, I can use the wildcards

Reply:
For me, it's ok to use explicit imports, but it's not consistent within the PR

for example - https://github.com/crowdin/crowdin-api-client-java/pull/129/files#diff-cb949cfdae22c9d60840ccaa6e9ca4ab0562d19334658a2df8d28862a64f46f9R10 

I would suggest making it consistent in both of these places within the current PR

Reply:
Oh you are right, I didn't notice that. Fixed 🚀